resources/sass/_base.scss

```diff
@@ -1,47 +1,47 @@
 // Variables for themes to override
-$spacing: 4 !default;
-$color_size: 16 !default;
+:root {
+  --spacing: 4px;
```

Comment:
Right? It was also my initial assumption, however upon further look it appears that this was just done to make use of Sass calculation.

It makes sense to explicitly set the unit to `px` here rather than everywhere else in the CSS rewrite, correct? Also, a theme might use another unit of their choosing.

Using CSS' native `calc()` doesn't care _what_ the initial unit is, it just performs multiplication or division on it, which is our goal here.

Reply:
I think the intent was to force the unit to be pixels (Since if you use something like em in a recursive structure you'll get ballooning sizes)

Also for line-height there's merit to unitless input (`line-height: 1.1` is roughly `line-height: calc(font-size * 1.1)`) but again we don't actually use it. I have no strong opinions one way or the other

Reply:
I agree, also there is the possibility of not using `line-height` for this and make use of other approaches.

Reply:
The dom structure needs a rework anyway (It's too deep with too many elements and makes the browser take longer to render the dom than kint takes to render the markup) but that's a big change that touches everything including JS so it's definitely not on the table rn.